src/main/java/Duke.java

```diff
@@ -96,6 +96,7 @@ public DukeEmptyTaskException (String msg) {
     public static String parse_date(String s) {
```

Comment:
Instead of "parse_date", perhaps you could rename it to "parseDate", which conforms to the coding standard?